TVOAppliance/hf_upgrade.sh

```diff
@@ -95,6 +95,31 @@ function reconfigure_s3_service_path()
 	sed  -i "s~$src_string~$dest_string~g" $file_name
 
 }
+
+#function to keep only tls1_2 access
+function limit_access_to_tls1_2()
+{
+	config_file="/etc/nginx/nginx.conf"
+	config_line="ssl_protocols TLSv1.2;"
+
+	# Check if the line already exists in the file
+	if grep -q "$config_line" "$config_file"; then
+		echo "Line already exists in $config_file"
+	else
+	  # Add the line to the http section
+	  	sed -i '/^\s*http {/a \    ssl_protocols TLSv1.2;' "$config_file"
+	  	echo "Added line to $config_file"
+	  	pcs resource restart lb_nginx-clone
```

Comment:
don't restart it here, move it to the section below where we check if the appliance is already configured.
restart the pcs resource in that section and mention the resource name in the output msg.

Reply:
moved